apps/devhub/tests/test_views.py

```diff
@@ -1190,6 +1190,28 @@ def test_create_new_credentials(self):
         mock_creator.assert_called_with(self.user)
         self.assertRedirects(response, self.url)
 
+    def test_retreive_credentials(self):
+        patch = mock.patch('api.models.APIKey.get_jwt_key')
+        with patch as mock_creator:
+            mock_creator.return_value = APIKey(
```

Comment:
when you have to mock the return value for something, it's worth considering not mocking it. I think in this case it would be fine to not use a mock. You would just need to [create an APIKey](https://github.com/Nolski/olympia/blob/api-key-reload/apps/devhub/tests/test_views.py#L1176) like the other tests do.